sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -876,6 +876,16 @@ def validate(self, validator):
 
     return errors
 
+  def lookup_dataflow_service_option(self, key, default=None):
```

Comment:
I would prefer lookup_dataflow_service_option have the same implementation w.r.t. defaults as lookup_experiment to be consistent. You have changed the logic. If preexisting logic doesn't work for you, let's name it as a specialized a helper function   `get_cloud_debugger_service_name`.

Reply:
also, let's add unit tests for this helper.